tracing-log/README.md

```diff
@@ -0,0 +1,61 @@
+# tracing-log
+
+**Warning: Until `tracing-log` has a 0.1.0 release on crates.io, please treat every release as potentially breaking.**
+
+[`log`] compatibility for [`tracing`].
+
+[![Crates.io][crates-badge]][crates-url]
+[![Documentation][docs-badge]][docs-url]
+[![MIT licensed][mit-badge]][mit-url]
+[![Build Status][azure-badge]][azure-url]
+[![Gitter chat][gitter-badge]][gitter-url]
+![maintenance status][maint-badge]
+
+[Documentation][docs-url] |
+[Chat][gitter-url]
+
+[crates-badge]: https://img.shields.io/crates/v/tracing-log.svg
+[crates-url]: https://crates.io/crates/tracing-log
+[docs-badge]: https://docs.rs/tracing-log/badge.svg
+[docs-url]: https://docs.rs/tracing-log
+[mit-badge]: https://img.shields.io/badge/license-MIT-blue.svg
+[mit-url]: LICENSE
+[azure-badge]: https://dev.azure.com/tracing/tracing/_apis/build/status/tokio-rs.tracing?branchName=master
+[azure-url]: https://dev.azure.com/tracing/tracing/_build/latest?definitionId=1&branchName=master
+[gitter-badge]: https://img.shields.io/gitter/room/tokio-rs/tracing.svg
+[gitter-url]: https://gitter.im/tokio-rs/tracing
+[maint-badge]: https://img.shields.io/badge/maintenance-experimental-blue.svg
+
+## Overview
+
+[`tracing`] is a framework for instrumenting Rust programs with context-aware,
+structured, event-based diagnostic information. This crate provides
+compatibility layers for using `tracing` alongside the logging facade provided
+by the [`log`] crate.
+
+This crate provides:
+
+- [`LogTracer`], a [`log::Log`] implementation that consumes [`log::Record`]s
+  and outputs them as [`tracing::Event`].
+- [`TraceLogger`], a [`tracing::Subscriber`] implementation that outputs
```

Comment:
I'm not clear on what the difference between a LogTracer is and a TraceLogger. At least not from reading this, the definitions given here make them sound identical.

Reply:
is https://github.com/tokio-rs/tracing/pull/266/commits/2c6864f5dfb74327fe31d85e36a566a4b384073f any clearer?

Reply:
yup! that what I assumed was meant and its no longer ambiguous. I'm quite satisfied with this resolution.